### DISCUSSION TITLE
Discord Invite Code?

### DISCUSSION THREAD

**Discussion by theKlois** [bridge-core/bridge.#325] [General]
Pardon me if I am missing an obvious discord link somewhere. I did see it in the program but I am currently on my phone and not able to use my computer till later. 

Would anyone be able to post the invite code or message me with it?

Thank you!

**Comment by Chikorita-Lover** [comment]
Hello, here is an invite I just created that expires in a week: [https://discord.gg/BzjuaVPK](https://discord.gg/BzjuaVPK)

**Reply by theKlois** [reply]
Thank you!
